rules/explicitOverrideRule.ts

```diff
@@ -177,6 +182,8 @@ class Walker extends Lint.AbstractWalker<IOptions> {
             // But no base symbol was found
             (base === undefined || base.baseClass === undefined && base.baseInterface === undefined)
         ) {
+            // TODO: When given two @override decorators following each other (with a new line or a whitespace between them),
+            // the fixer does not break the line
```

Comment:
What do you mean exactly? That doesn't sound like something we should worry about.

Reply:
Please take a look at the test files of both decorator and jsdoc tags.
```javascript
@override
@override function bar2(): void { };
```
becomes
```javascript
@override function bar2(): void { };
```
but it should be
```javascript
@override
function bar2(): void { };
```

Reply:
I see

Reply:
So the todo stays, it doesn't seem like an urgent issue though